src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1120,6 +1153,10 @@ codet java_bytecode_convert_methodt::convert_instructions(
     }
   }
 
+  // Clean the list of temporary variables. These are local variables in the
+  // goto function used to represent temporary values of the JVM operand stack?
```

Comment:
Also other things such as temporaries to hold a newly-allocated instance between allocation and a constructor being called. Any caller of `tmp_variable`, in short.